examples/resources/biganimal_cluster/ha/resource.tf

```diff
@@ -78,7 +78,23 @@ resource "biganimal_cluster" "ha_cluster" {
   read_only_connections = true
   region                = "us-east-1"
   superuser_access      = true
-  pgvector              = true
+  pgvector              = false
+
+  pg_bouncer = {
+    is_enabled = false
+    #  settings = [ # uncomment and set settings if is_enabled is true
```

Comment:
If is_enabled is true, remove the comment and enter the settings. Should you prefer something different from the defaults.